### PR TITLE
1.2.0

### PR DESCRIPTION
**Pull request by M-Porter**
**Changes**
* Styles now consistent across elements
* Naming conventions consistent
* Erratic behavior with elements in the content areas fixed
* Dotted border now 2px so that the dots are actually visible

### CODE REVIEW

File: files/css/element.css [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
 
-/*= 3D
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
-/* Base for shadow mixins */
-.s3d-shadow {
-    box-shadow: @s3d-shadow-horizontal @s3d-shadow-vertical @s3d-shadow-blur @s3d-shadow-spread @s3d-shadow-color;
+    &--material {
+        .callout-base;
+        .material;
+    }
```
Comment (puddi) [inline files/css/element.css]: Alternatively:

```less
.callout-box {
    .callout-base;
    
    &--material {
        .material;
    }
}

**Comment by puddi** [comment]
:+1:

File: files/css/element.css (hunk 1 above)
Comment (M-Porter) [inline files/css/element.css]: yeah i've been using BEM standards for my naming conventions.

**Comment by M-Porter** [comment]
screen shot 2015-10-07 at 11 24 52 am
screen shot 2015-10-07 at 11 24 46 am
screen shot 2015-10-07 at 11 25 34 am

Live screenshots since i don't have live links.

**Pull request merged**